packages/medusa/src/services/product-category.ts

```diff
@@ -27,6 +27,39 @@ class ProductCategoryService extends TransactionBaseService {
     this.productCategoryRepo_ = productCategoryRepository
   }
 
+  /**
+   * Lists product category based on the provided parameters and includes the count of
+   * product category that match the query.
+   * @return an array containing the product category as
+   *   the first element and the total count of product category that matches the query
+   *   as the second element.
+   */
+  async listAndCount(
+    selector: QuerySelector<ProductCategory>,
+    config: FindConfig<ProductCategory> = {
+      skip: 0,
+      take: 100,
+      order: { created_at: "DESC" },
+    }
+  ): Promise<[ProductCategory[], number]> {
+    const manager = this.manager_
```

Comment:
todo: this.transactionManager_ ?? this.manager_

Reply:
good eye! ;)